client/asset/dcr/spv.go

```diff
@@ -773,7 +774,7 @@ func (w *spvWallet) SyncStatus(ctx context.Context) (bool, float32, error) {
 		return false, 0, nil
 	}
 
```

Comment:
Related, above we use `w.dcrWallet.MainChainTip` and `bestPeerInitialHeight`, but should we be using this?

```go

// EstimateMainChainTip returns an estimated height for the current tip of the
// blockchain. The estimate is made by comparing the initial height reported by
// all connected peers and the wallet's current tip. The highest of these values
// is estimated to be the mainchain's tip height.
func (s *Syncer) EstimateMainChainTip() int32 {
	_, chainTip := s.wallet.MainChainTip(context.Background())
	s.forRemotes(func(rp *p2p.RemotePeer) error {
		if rp.InitialHeight() > chainTip {
			chainTip = rp.InitialHeight()
		}
		return nil
	})
	return chainTip
}
```

It's not clear why we have the `bestPeerInitialHeight()` helper defined given

Reply:
Thought about making that change but it's basically the same thing anyways, so decided against it.

My next commit will cap progress percentage at 99.9% while synced == false.